packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -341,6 +342,7 @@ const Select = <
           text: option.text,
           value,
           type: option.type === "heading" ? "heading" : "option",
+          useLangAttr: option.useLangAttr,
```

Comment:
We should name this with an `is` or `has` prefix if it's a boolean and not use shorthand, so it matches other props in Odyssey.

`hasLanguageAttribute` works. But I'm curious, why a boolean? Don't we want to pass this as a string? Why have the language be the `option`'s `value`?

Reply:
The value of language option item can be used for lang attribute directly, like 'en', 'fr'. 

But I think what you mentioned above makes sense, giving `lang` directly in props can give the downstream app better flexibility.

Reply:
updated by using `lang` prop